toolchains/prebuilt_toolchains.py

```diff
@@ -166,6 +172,7 @@
 }
 
 NINJA_VERSIONS = (
+    "1.12.1",
```

Comment:
Is the update to the ninja repo list missing for this update?

Reply:
oh, I guess it is; when I run the script by hand and try to commit the result, the repo hooks don't let me; when I let the repo hooks run the script themselves; it looks like they don't stage the ninja file, just the cmake file

Reply:
Oh, I see what's happening. The hook requires using buildifier 6.1.0, but my buildifier is 7.1.2, and they produce different output. PR updated.